cmd/integration/commands/stage_stages_zkevm.go

```diff
@@ -101,6 +95,8 @@ func unwindZk(ctx context.Context, db kv.RwDB) error {
 		return err
 	}
 
+	stages.SaveStageProgress(tx, stages.HighestSeenBatchNumber, unwindBatchNo)
```

Comment:
Good point, and I think this is unnecessary now, because it will be covered by unwind in sequencing stage [here](https://github.com/0xPolygonHermez/cdk-erigon/blob/f911a0daf793b77fa03da3b4f57838630ffe42c6/zk/stages/stage_sequence_execute_unwind.go#L64).